dashboard/app/assets/javascripts/editor/json_editor.js

```diff
@@ -0,0 +1,106 @@
+/* global CodeMirror */
+
+function calloutEditor(container, options) {
+
```

Comment:
nit: extra blank line

also, shoudl this be function jsonEditor instead of calloutEditor?